src/main/java/Deadline.java

```diff
@@ -0,0 +1,22 @@
+public class Deadline extends Task{
+
+    private static String by;
+
+    public Deadline(String description, String by){
```

Comment:
You could leave a space here before `{` for layout consistency. This applies to other similar parts of your code which does the same (e.g. line 11 and 15).